packages/ember-testing/tests/acceptance_test.js

```diff
@@ -364,7 +360,7 @@ QUnit.test('test must not finish while asyncHelpers are pending', function () {
   }
 });
 
-QUnit.test('visiting a URL that causes another transition should yield the correct URL', function () {
+test('visiting a URL that causes another transition should yield the correct URL', function () {
```

Comment:
These need to be skipped now because `{{input}}` is not finished on the glimmer side.